production/loki-mixin/config.libsonnet

```diff
@@ -4,19 +4,22 @@
 
   _config+:: {
     // Tags for dashboards.
-    tags: ['loki'],
+    tags: ['loki', 'telemetry'],
 
     // The label used to differentiate between different application instances (i.e. 'pod' in a kubernetes install).
-    per_instance_label: 'pod',
+    per_instance_label: 'k8s_pod_name',
```

Comment:
i think we should leave these as their previous defaults for now and revisit this as a separate discussion/PR. 

We are going to be making a lot of changes to prefer opentelemtry by default and I think this would make sense once we get a bit farther down that path